azure/flake.nix

```diff
@@ -0,0 +1,125 @@
+{
+  description = "DevShell with TPM-based Azure login and OpenSSL engine";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.05";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils, ... }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs { inherit system; };
+
+        enginePath = "/var/lib/jenkins/workspace/GhafSB-test/e_akv.so";
```

Comment:
TODO: This needs a proper location instead of being hardcoded to a jenkins WS ...